lib/swig/includes.i

```diff
@@ -0,0 +1,7 @@
+//Apply strictly to python
```

Comment:
@stdevHan is this specific to Python ?

Reply:
@olemis  there are only things that need language exclusion